src/libraries/Microsoft.PowerFx.Core/Types/DTypeSpecParser.cs

```diff
@@ -11,8 +11,8 @@ namespace Microsoft.PowerFx.Core.Types
 {
     internal static class DTypeSpecParser
     {
-        private const string _typeEncodings = "?ebnshdipmgo$cDT!*%lLNZPQqVOX-w";
-        private static readonly DType[] _types = new DType[]
+        internal const string _typeEncodings = "?ebnshdipmgo$cDT!*%lLNZPQqVOX-w";
```

Comment:
>internal [](http://example.com/codeflow?start=8&length=8)

Why are we increasing visiblity? Is that related to 2328, or another fix?

Reply:
This is used in TestUtils.cs at lines 342-343 in order to implement DT2 helper method which can now support display names and allows the test update in TestDelegationValidation.cs to work as expected.